conda/common/logic.py

```diff
@@ -621,12 +622,10 @@ def ExactlyOne(self, vals, polarity=None, name=None):
         return self.Eval_(what, (vals,), polarity, name)
 
     def LB_Preprocess_(self, equation):
-        if type(equation) is dict:
```

Comment:
intentional to kill the dict equation possibility?

Reply:
Yes, intentional. `LB_Preprocess` is moved to `_logic.Clauses` in a latter commit and that class doesn't use a `dict` anymore. The user-facing part (`logic.Clauses.minimize` and `logic.Clauses.LinearBound`) still allow the `dict` input and convert it accordingly.